taqueria-plugin-taquito/fund.ts

```diff
@@ -0,0 +1,94 @@
+import {
+	getCurrentEnvironment,
+	getCurrentEnvironmentConfig,
+	sendAsyncErr,
+	sendJsonRes,
+	sendWarn,
+} from '@taqueria/node-sdk';
+import { Environment } from '@taqueria/node-sdk/types';
+import { TezosToolkit } from '@taquito/taquito';
+import {
+	configureToolKitWithNetwork,
+	FundOpts as Opts,
+	getDeclaredAccounts,
+	getNetworkInstantiatedAccounts,
+	getNetworkWithChecks,
+} from './common';
+import { performTransferOps, TableRow } from './transfer';
+
+const configureTezosToolKit = (parsedArgs: Opts, env: Environment.t): Promise<TezosToolkit> => {
```

Comment:
This function seems redundant with `configureTezosToolkitWithNetwork` / `configureTezosToolkitWithSandbox` from common.ts - can we consolidate?